django/template/engine.py

```diff
@@ -74,18 +73,14 @@ def get_default():
         # local imports are required to avoid import loops.
         from django.template import engines
         from django.template.backends.django import DjangoTemplates
-        django_engines = [engine for engine in engines.all()
-                          if isinstance(engine, DjangoTemplates)]
-        if len(django_engines) == 1:
-            # Unwrap the Engine instance inside DjangoTemplates
-            return django_engines[0].engine
-        elif len(django_engines) == 0:
-            raise ImproperlyConfigured(
-                "No DjangoTemplates backend is configured.")
-        else:
-            raise ImproperlyConfigured(
-                "Several DjangoTemplates backends are configured. "
-                "You must select one explicitly.")
+        try:
+            # Return the Engine instance of the first DjangoTemplates.
+            return next(
+                engine for engine in engines.all()
+                if isinstance(engine, DjangoTemplates)
+            ).engine
+        except StopIteration:
+            raise ImproperlyConfigured('No DjangoTemplates backend is configured.')
```

Comment:
I'm not sure this `next()` dance is really required, I find the following much easier to read

```python
for engine in engines.all():
    if isinstance(engine, DjangoTemplates):
        return engine.engine
raise ImproperlyConfigured('No DjangoTemplates backend is configured.')
```